lib/block_orthogonalize.cu

```diff
@@ -72,6 +72,7 @@ namespace quda {
         strcat(aux, geo_str);
         if (d < V.Ndim() - 1) strcat(aux, "x");
       }
+      if (geoBlockSize == 1) errorQuda("Invalid MG aggregate size %d", geoBlockSize);
```

Comment:
While it's certainly outside the scope of this PR, it could be interesting to add support for this in the future, thinking in the context of "thinning" degrees of freedom (reducing Nc) at fixed lattice volume. I can't see why that'd ever be worth it, but it's not strictly irrelevant.